platforms/documentation/docs/src/docs/userguide/reference/plugin-development/reporting_problems.adoc

```diff
@@ -13,66 +13,143 @@
 // limitations under the License.
 
 [[sec:reporting_problems]]
-= Reporting problems
+= Reporting Plugin Problems with the Problems API
 :keywords: problems, api, problem
 
-Plugins can report problems through Gradle's problems-reporting APIs.
-The APIs report rich, structured information about problems happening during the build.
-This information can be used by different user interfaces such as Gradle's console output, Build Scan, or IDEs to communicate problems to the user in the most appropriate way.
+Gradle’s Problems API lets plugins report rich, structured information about issues that occur during a build.
 
-The following example shows an issue reported from a plugin:
+This data powers Gradle’s console output, Build Scans, and IDE integrations so users see clear, actionable problem reports.
```

Comment:
Just a quick clarification. The problem reports only show up on the console if they fail the build. We'll soon add simple console rendering but it's not merged yet.

Reply:
Added "failing build".